core/src/main/java/org/polypheny/db/plan/volcano/VolcanoCost.java

```diff
@@ -105,6 +105,12 @@ public boolean isInfinite() {
     }
 
 
+    @Override
+    public double getCosts() {
+        return getRows() + getCpu() + getIo();
```

Comment:
@isabelge Please remember this method. We might need some weights here in the future. Simply summing up the three costs is probably not an optimal solution. However, it should be fine for this PR.